### PR TITLE
DDP-6730 faq and participation pages redesign

### PR DESCRIPTION
**Pull request by aglushko-broad**

### CODE REVIEW

File: ddp-workspace/projects/ddp-pancan/src/app/components/participation/participation.component.scss [hunk 1]
```diff
@@ -39,11 +40,17 @@
...
         }
 
         &__text {
-            margin-bottom: 2.13rem;
+            margin: 0.5rem 0 2.13rem;
+            font-size: 1.5rem;
+            line-height: 2.125rem;
         }
 
         &__btn {
```
Comment (alextanlan) [inline ddp-workspace/projects/ddp-pancan/src/app/components/participation/participation.component.scss:55]: suggest adding 
`font-family: $primary-font-medium;`

buttons are bold on mockup

**Comment by alextanlan** [comment]
I anticipate that the main title background should grab all width. (see `Cómo participar`)
Can you please check it out ?

c2

**Comment by alextanlan** [comment]
The content doesn't fit small screen:
c1

**Comment by alextanlan** [comment]
The title is extensive.
d2

File: ddp-workspace/projects/ddp-pancan/src/app/components/participation/participation.component.scss (hunk 1 above)
Comment (aglushko-broad) [reply ddp-workspace/projects/ddp-pancan/src/app/components/participation/participation.component.scss:55]: done

**Comment by aglushko-broad** [comment]
> I anticipate that the main title background should grab all width. (see `Cómo participar`)
> Can you please check it out ?
> 
> c2

thanks! fixed

**Review by alextanlan — APPROVED** [review]

**Comment by aglushko-broad** [comment]
> The content doesn't fit small screen:
> c1

fixed, though the problem is not related to my PR

**Comment by aglushko-broad** [comment]
> The title is extensive.
> d2

fixed

**Pull request merged**